tests/testthat/test-organizations.R

```diff
@@ -1,3 +1,8 @@
+vcr::use_cassette("organizations", {
 test_that("gh_organizations works", {
-  testthat::expect_type(gh_organizations(), "character")
+
+  testthat::expect_message(foo <- gh_organizations(), "Retrying")
```

Comment:
see @sckott it's a case where I think it's easier to just wrap the whole test with use_cassette(). :thinking:

Reply:
Note: when one wraps the whole test in `use_cassette()`, there's no straightforward way to "skip" the test.